tests/Storage/ExampleStorage.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+namespace Shepard\Tests\Storage;
+
+use Shepard\Storage\StorageInterface;
+use Symfony\Component\Process\Process;
+
+class ExampleStorage implements StorageInterface
```

Comment:
ja, és nem csak exemple storage kell, hanem kell egy rendes implementáció, legalább a local, de jó, ha van S3 is.